tests/test_config.py

```diff
@@ -66,3 +89,64 @@ def test_ssl_config(certfile_and_keyfile):
     config.load()
 
     assert config.is_ssl is True
+
+
+@pytest.mark.parametrize(
+    "use_colors, expected",
+    [(None, None), (True, True), (False, False), ("invalid", False)],
+)
```

Comment:
can we use ids kwargs on this parametrized test to get a sense of what it means ?

Reply:
sure thing. i'll give that a shot

Reply:
excellent !